frontend/src/css/modules/submission.scss

```diff
@@ -7,3 +7,11 @@
 .time {
 	position: static;
 }
+
+.pointer {
```

Comment:
I think there is already a class present named pointer. Please recheck again in base.scss.